wagtailmenus/models.py

```diff
@@ -106,6 +106,11 @@ class MenuItem(models.Model):
         blank=True,
         help_text=_("Must be set if you wish to link to a custom URL."),
     )
+    handle = models.CharField(
+        max_length=100,
+        blank=True,
+        help_text=_(
+            "Used to identify a specific menu item in the template code."))
```

Comment:
One very last thing, and then we're done :) Could you alter this help text to read more like the README text? maybe something like...

> Use this field to optionally specify an additional value for each menu item, which you can then reference in custom menu templates.